### PR TITLE
Clean up unnecessary preview headers

### PR DESCRIPTION
**Pull request by n1lesh**
Removes unnecessary **mediaTypeRequiredApprovingReviewsPreview** headers as described in #1779

### CODE REVIEW

**Review by gmlewis — APPROVED** [review]
Thank you, @n1lesh !
LGTM.

Awaiting second LGTM before merging.

**Review by wesleimp — APPROVED** [review]
LGTM 👌🏼

**Comment by gmlewis** [comment]
Thank you, @wesleimp !
Merging.

**Pull request merged**
